conferences/socrates-ca.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "SoCraTes Canada",
+  "url": "https://socrates-ca.github.io/",
+  "code-of-conduct": "https://socrates-ca.github.io/#portfolioModal2",
```

Comment:
This link doesn't show the code of conduct for me on mobile, not sure if it's possible to get a direct link?

Reply:
It should open a modal with values but it seems that it does not work. I opened an issue on our side to fix this: https://github.com/socrates-ca/socrates-ca.github.io/issues/5#issue-511921264

Reply:
@franziskas The issue is now fixed, thanks to @nicoespeon! You should bee able to see the code of conduct with the direct link in the PR.